homework/src/index.js

```diff
@@ -29,19 +29,100 @@
     });
     return elem;
   }
+  function Repositories(index, data, container) {
+    const repoInfo = createAndAppend('div', container, { 'class': 'left-div box' });
+    const table = createAndAppend('table', repoInfo, { 'id': 'info-table' });
+
+    const repoTr = createAndAppend('tr', table);
+    createAndAppend('td', repoTr, { 'text': 'Repository :', 'class': 'label' });
+    const repoName = createAndAppend('td', repoTr, { 'id': 'repo-name' });
+    const url = createAndAppend('a', repoName, { 'target': '_blank', });
+
+    const descriptionTr = createAndAppend('tr', table);
+    const descriptionName = createAndAppend('td', descriptionTr, { 'text': 'Description :', 'class': 'label' });
+    const descriptionDetails = createAndAppend('td', descriptionTr);
+
+    const forkTr = createAndAppend('tr', table);
+    createAndAppend('td', forkTr, { 'text': 'Forks :', 'class': 'label' });
+    const forkNum = createAndAppend('td', forkTr);
+
+    const updateTr = createAndAppend('tr', table);
+    createAndAppend('td', updateTr, { 'text': 'Updated :', 'class': 'label' });
+    const updateNum = createAndAppend('td', updateTr);
+
+    url.innerText = data[index].name;
+    url.setAttribute('href', data[index].html_url);
+    if (data[index].description === null) {
+      descriptionName.innerHTML = '';
+      descriptionDetails.innerText = '';
+    } else {
+      descriptionName.innerText = "Description :";
+      descriptionDetails.innerText = data[index].description;
+    }
+    forkNum.innerText = data[index].forks;
+    const updateRepo = new Date(data[index].updated_at);
+    updateNum.innerText = updateRepo.toLocaleString("en-US");
+  }
+
+  function Contributors(data, container) {
+    const contributorsDiv = createAndAppend('div', container, { 'class': 'right-div box' });
+    createAndAppend('p', contributorsDiv, { 'text': 'Contributions', 'class': 'contributions' });
+    const ul = createAndAppend('ul', contributorsDiv);
+    data.forEach(contributor => {
+      const li = createAndAppend('li', ul);
+      li.setAttribute('target', '_blank');
+      li.addEventListener("click", () => { window.open(contributor.html_url) });
```

Comment:
Why would you use an eventListener? Why not just ad an anchor element `<a href=contributor.html_url>`?